scripts/cardano-node.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+# Run a mainnet Cardano node.
+# Example usage:
+#
+# NODE_BIN_DIR=../cardano-node-bin \
+#   NODE_DIR=../cardano-node
+#   ./scripts/cardano-node.sh
+
+set -euo pipefail
+
+node_zipped="cardano-node-1.34.1-linux.tar.gz"
+node_config_files=(
+  "mainnet-config.json"
+  "mainnet-byron-genesis.json"
+  "mainnet-shelley-genesis.json"
+  "mainnet-alonzo-genesis.json"
+  "mainnet-topology.json"
+)
+
+trap 'kill $cardano_node_pid; exit' INT TERM QUIT
+
+set -x
+mkdir -p "$NODE_BIN_DIR"
+# Download cardano-node binary
+wget -nc https://hydra.iohk.io/build/13065769/download/1/$node_zipped -P "$NODE_BIN_DIR"
+
+tar zxvf "$NODE_BIN_DIR"/$node_zipped -C "$NODE_BIN_DIR"
+
+mkdir -p "$NODE_DIR"
+# Download config files
+for x in "${node_config_files[@]}"; do
+  wget -nc https://hydra.iohk.io/build/7370192/download/1/"$x" -P "$NODE_DIR"
+done
+
+# Launch node
```

Comment:
I think we should separate the downloading and running parts. There's no reason to download the node every single time if we're just going to run it on a machine that already has it.

Reply:
Sounds like you need some comments :)